README.md

```diff
@@ -200,4 +200,4 @@ Build and test orchestration is performed by [Jenkins][21].
 [20]:   https://tessel.io/
 [21]:   https://jenkins.io/
 [pivotal]: https://www.pivotalagency.com.au/
-[securo]: http://securogroup.com/
\ No newline at end of file
+[securo]: http://securogroup.com/
```

Comment:
looks like it is adding one. I used the GitHub ui for this, likely added the new line